package.json

```diff
@@ -37,20 +37,20 @@
     "@types/dotenv": "4.0.3",
     "@types/enzyme": "3.1.14",
     "@types/enzyme-adapter-react-16": "1.0.3",
-    "@types/jest": "23.3.9",
+    "@types/jest": "24.0.6",
     "@types/webpack": "4.4.17",
     "@types/webpack-bundle-analyzer": "2.13.0",
     "@types/webpack-node-externals": "1.6.3",
-    "apollo": "2.1.8",
+    "apollo": "^2.5.1",
```

Comment:
I had to upgrade this because I was running into https://github.com/apollographql/apollo-tooling/issues/962.